app/src/main/java/io/pslab/activity/RoboticArmActivity.java

```diff
@@ -318,10 +318,18 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
                         editEnter = true;
                     }
                 }
+                degreeText1.clearFocus();
                 return false;
             }
         });
 
+        degreeText1.setOnFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View v, boolean hasFocus) {
+                degreeText1.setCursorVisible(false);
```

Comment:
What if we make cursor visibility false in every other text? As an example; here it'll be:
degreeText1.setCursorVisible(true);
degreeText2.setCursorVisible(false);
degreeText3.setCursorVisible(false);
degreeText4.setCursorVisible(false);

Will it work? Just try it and see :thinking:

Reply:
But the cursor is already false for all other texts if I have clicked on one of them. The issue is when we don't press the Done button on the soft keyboard and so the focus stays on that one text and the cursor blinks continuously until the Done is pressed.

Reply:
`setOnFocusChangeListener` will be fired for only two views. One that is getting the focus and the one that is losing focus. Two textviews will not get this method called.